core/iwasm/interpreter/wasm_opcode.h

```diff
@@ -791,14 +791,15 @@ typedef enum WASMAtomicEXTOpcode {
 #define SET_GOTO_TABLE_ELEM(opcode) [opcode] = HANDLE_OPCODE(opcode)
 
 #if (WASM_ENABLE_JIT != 0 || WASM_ENABLE_FAST_INTERP != 0) \
-    && WASM_ENABLE_SIMD != 0
+    && WASM_ENABLE_SIMD != 0 && WASM_ENABLE_SIMDE != 0
```

Comment:
because of the workaround, https://github.com/bytecodealliance/wasm-micro-runtime/blob/WAMR-2.2.0/core/iwasm/interpreter/wasm_interp_classic.c#L6584C1-L6589C7, maybe we can simplify this to `#if WASM_ENABLE_SIMD !=0 || WASM_ENABLE_SIMDE != 0`